Source/Pawnmorphs/Esoteria/PawnmorpherModInit.cs

```diff
@@ -331,9 +332,25 @@ private static void TransferPatchesToExplicitRaces()
                 }
 
             }
+        }
 
+        private static void AddMutationsToWhitelistedRaces()
+        {
+            List<AlienPartGenerator.BodyAddon> allAddons = GetAllAddonsToAdd().ToList();
 
-
+            foreach (string raceDefName in PawnmorpherMod.Settings.visibleRaces)
+            {
+                ThingDef_AlienRace race = (ThingDef_AlienRace)ThingDef.Named(raceDefName);
```

Comment:
I believe this causes and error if there is no def with that name, we'll probably want to use DefDatabase.GetNamed with errrorOnFail = false and check for null ourselves. in case a mod in removed but the race def name still persists in the settings